squirrel.frontier/src/test/resources/seeds/file3.csv

```diff
@@ -0,0 +1,131 @@
+uri
```

Comment:
What is the difference between file1, 2 and 3?

Reply:
file 1 is the one correct, with URI and TYPE
file 2 is missing the Type for some URis
file 3 is missing the header